apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -768,29 +768,96 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
            end)
   end
 
+  describe "available piggybacks" do
+    @tag fixtures: [:alice, :processor_filled, :transactions]
+    test "detects available piggybacks because txs not seen in valid block",
+         %{alice: alice, processor_filled: processor, transactions: [tx1, tx2]} do
+      exit_processor_request = %ExitProcessor.Request{
+        blknum_now: 5000,
+        eth_height_now: 5
+      }
+
+      %Transaction{outputs: [%{owner: tx1_owner1}, %{owner: tx1_1owner2}]} = tx1
+
+      event_1 =
+        prepare_piggyback_available_event(
+          [%{address: alice.addr, index: 0}, %{address: alice.addr, index: 1}],
+          [
+            %{address: tx1_owner1, index: 0},
+            %{address: tx1_1owner2, index: 1},
+            %{address: Crypto.zero_address(), index: 2},
+            %{address: Crypto.zero_address(), index: 3}
+          ],
+          Transaction.encode(tx1)
+        )
+
+      %Transaction{outputs: [%{owner: tx2_owner1}, %{owner: tx2_1owner2}]} = tx2
+
+      event_2 =
+        prepare_piggyback_available_event(
+          [%{address: alice.addr, index: 0}, %{address: alice.addr, index: 1}],
+          [
+            %{address: tx2_owner1, index: 0},
+            %{address: tx2_1owner2, index: 1},
+            %{address: Crypto.zero_address(), index: 2},
+            %{address: Crypto.zero_address(), index: 3}
+          ],
+          Transaction.encode(tx2)
+        )
+
+      assert {:ok, [^event_1, ^event_2]} = exit_processor_request |> Core.invalid_exits(processor)
+    end
+
+    @tag fixtures: [:alice, :processor_filled, :transactions]
+    test "don't detects available piggybacks because txs seen in valid block", %{
```

Comment:
doesn't detect piggybacks.

Besides that, I'd include just one transaction in the block and make sure, that the one that gets reported is the one that **wasn't** included